tests/hwloc/ports/include/rsmi/rocm_smi.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2013 Inria.  All rights reserved.
+ * Copyright (c) 2020, Advanced Micro Devices, Inc. All rights reserved.
+ * Written by Advanced Micro Devices,
+ * See COPYING in top-level directory.
+ */
+
+#ifndef HWLOC_PORT_RSMI_RSMI_H
+#define HWLOC_PORT_RSMI_RSMI_H
+
+/* we need to replace any RSMI-related #define that configure may have put in private/autogen/config.h */
+#ifndef HWLOC_CONFIGURE_H
+#error rsmi.h must be included after private/autogen/config.h
+#endif
```

Comment:
This #error should say rocm_smi.h instead of rsmi.h

Reply:
Ok, good finding

Reply:
typo in rsmi_smi.h :)